spec/std/http/server/server_spec.cr

```diff
@@ -8,8 +8,8 @@ private class RaiseErrno
   include IO
 
   def read(slice : Bytes)
-    Errno.value = @value
-    raise Errno.new "..."
+    Crystal::System::Errno.value = @value
+    raise OSError.create "..."
```

Comment:
What?! Using `Crystal::System::Errno` makes no sense: it's not a platform abstraction :/

Using a ... error message is also, huh...

Reply:
Furthermore pushing `Errno` as `Crystal::System::Errno` means that developers of C bindings will now need to use `Crystal::System`... but developers musn't.

Reply:
Well with the namespace I guess I see what you mean. But Crystal cannot provide access to `errno` in a cross-platform way because it's a UNIX-specific thing, and at the same time, `Crystal::System::Errno` **is**, in fact, a platform abstraction (different POSIX platforms). What do you suggest?
By the way, developers technically don't have to access `Crystal::System::Errno`, they can just use `OSError.create.errno`, but they usually shouldn't need to.

Reply:
Reworked this. But now accessing `errno` is a compilation error on non-POSIX.
at least I think so...